integr8ly/cloud-resource-operator
# set backup retention settings for elasticache instances to 31 days

enure the setting matches rds and is based on [1].

verification:
- run the operator locally
- seed a managed redis cr
- wait for an elasticache instance to provision successfully
- ensure backup retention on the elasticache instance is 31 daysset backup retention settings for elasticache instances to 31 days

enure the setting matches rds and is based on [1].

verification:
- run the operator locally
- seed a managed redis cr
- wait for an elasticache instance to provision successfully
- ensure backup retention on the elasticache instance is 31 days

[1] https://issues.redhat.com/browse/INTLY-3419?focusedCommentId=13786395&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13786395
